zeal/rpc/rpc.go

```diff
@@ -99,8 +107,27 @@ func (s *server) FetchBlob(ctx context.Context, req *pb.FetchBlobRequest) (*pb.F
 	if ar, err := s.storageClient.CheckActionCache(ctx, dg.ToProto()); err != nil {
 		log.Error("Failed to check action cache: %s", err)
 	} else if ar != nil {
+		if s.forceCasCheck {
+			if resp, err := s.storageClient.FindMissingBlobs(ctx, &rpb.FindMissingBlobsRequest{
+				InstanceName: s.storageClient.InstanceName,
+				BlobDigests:  []*rpb.Digest{dg.ToProto()},
```

Comment:
I don't think this works? The digest is the digest of the message, which we're using as an arbitrary key into the AC. I don't think we expect it to exist in the CAS.

Reply:
Updated in the latest, we can assume we have exactly 1 output file, so just grab it's digest.